models/public/colorization-v2-norebal/colorization-v2-norebal.md

```diff
@@ -0,0 +1,109 @@
+# colorization-v2-norebal
+
+## Use Case and High-Level Description
+
+The `colorization-v2-norebal` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+group of models designed to perform image colorization. For details
+about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+This model differs from model `colorization-v2` in that metrics did not take into account
+balancing of rare classes during training.
+
+Model consumes as input L-channel of LAB-image.
+Model give as output predict A- and B-channels of LAB-image.
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Colorization  |
+| GFLOPs            | -             |
+| MParams           | -             |
+| Source framework  | Caffe\*       |
+
+## Accuracy
+
```

Comment:
we need expected accuracy metrics on validation dataset

Reply:
@vladimir-dudnik corrected